public_html/wp-content/plugins/wordcamp-attendee-survey/includes/cron.php

```diff
@@ -0,0 +1,170 @@
+<?php
+
+namespace WordCamp\AttendeeSurvey\Cron;
+
+defined( 'WPINC' ) || die();
+
+use CampTix_Plugin;
+
+use function WordCamp\AttendeeSurvey\Email\{get_email_id, queue_survey_email};
+use function WordCamp\AttendeeSurvey\Page\{publish_survey_page};
+
+/**
+ * Constants.
+ */
+const DAYS_AFTER_TO_SEND = 2;
+
+/**
+ * Actions & hooks
+ */
+add_action( 'init', __NAMESPACE__ . '\schedule_jobs' );
+add_action( 'wc_attendee_survey_email', __NAMESPACE__ . '\queue_attendee_survey' );
+
+
+/**
+ * Logs a message to the CampTix email log.
+ */
+function log( $message, $post_id, $data = null ) {
+	/* @var CampTix_Plugin $camptix */
+	global $camptix;
+
+	$camptix->log( $message, $post_id, $data );
+}
+
+/**
+ * Add cron jobs to the schedule.
+ */
+function schedule_jobs() {
+	if ( ! wp_next_scheduled( 'wc_attendee_survey_email' ) ) {
+		wp_schedule_event( time(), 'hourly', 'wc_attendee_survey_email' );
```

Comment:
> Does it make sense to run the cron job like this? Should we schedule a single event?

I think this is fine. The downside is that it runs all the time, even when it's not needed, but the early returns mitigate the performance impact of that for the most part.

That can also be a benefit. It could be fragile to only schedule it once, since the camp date might change after it's initially scheduled, or something could fail the first time it tries to legitimately queue it, etc.